pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Optional: add Maven wrapper. That can be useful for both local development and CI/CD.

Optional: it is not the part of the task, but to keep code clean do the following:
1. Add [.editorconfig](https://editorconfig.org/) file. Intellij idea can generate it.
2. Add code analyzer [e.g. pmd](https://pmd.github.io/).
3. Add code coverage [e.g. jacoco](https://www.eclemma.org/jacoco/).